pyvista/plotting/plotter.py

```diff
@@ -5433,35 +5552,35 @@ def remove_scalar_bar(self, *args, **kwargs) -> None:  # numpydoc ignore=PR01,RT
 
     def add_point_labels(
         self,
-        points,
-        labels,
-        italic=False,
-        bold=True,
-        font_size=None,
-        text_color=None,
-        font_family=None,
-        font_file=None,
-        shadow=False,
-        show_points=True,
-        point_color=None,
-        point_size=None,
-        name=None,
-        shape_color='grey',
-        shape='rounded_rect',
-        fill_shape=True,
-        margin=3,
-        shape_opacity=1.0,
-        pickable=False,
-        render_points_as_spheres=False,
-        tolerance=0.001,
-        reset_camera=None,
-        always_visible=False,
-        render=True,
-        justification_horizontal=None,
-        justification_vertical=None,
-        background_color=None,
-        background_opacity=None,
-    ):
+        points: Sequence[float] | NumpyArray[float] | pyvista.DataSet | _vtk.vtkAlgorithm,
```

Comment:
Can we use `MatrixLike[float]` here?
```suggestion
        points: MatrixLike[float] | pyvista.DataSet | _vtk.vtkAlgorithm,
```